File: backend/src/main/java/com/atl/bakuboutiquehub/dto/SignupRequest.java
package com.atl.bakuboutiquehub.dto;

import lombok.Data;
import java.util.Set;

@Data
public class SignupRequest {
    private String email;
    private String password;
    private String phoneNumber;
    private String fullName;
    private String gender;
    private Integer age;
    private String referralSource;
    private Set<String> roles;
}

File: app/src/main/java/com/atl/bakuboutiquehub/network/model/SignupRequest.kt
package com.atl.bakuboutiquehub.network.model

data class SignupRequest(
    val email: String,
    val password: String,
    val fullName: String,
    val phoneNumber: String,
    val gender: String? = null,
    val age: Int? = null,
    val referralSource: String? = null,
    val roles: Set<String>? = null
)